src/core/modules.c

```diff
@@ -289,8 +289,9 @@ void modules_deinit(void)
 
 	while (list != NULL) {
 		module_uniq_destroy(list->data);
-		g_free(list->data);
+		gconstpointer tmp = list->data;
 		list = g_slist_remove(list, list->data);
+		g_free(tmp);
```

Comment:
`g_free` doesn't like the `gconstpointer`

Reply:
it's a bug in the C language

Reply:
Not the only one and not the worst one but still :)